Formula/kopia.rb

```diff
@@ -0,0 +1,48 @@
+class Kopia < Formula
+  desc "Fast and secure open-source backup"
+  homepage "https://kopia.io"
+  url "https://github.com/kopia/kopia.git",
+      tag:      "v0.10.4",
+      revision: "3447f1865f0b0536653aa6e9c07896ce73c3e600"
+  license "Apache-2.0"
+
+  depends_on "go" => :build
+
+  def install
+    ldflags = %W[
+      -s -w
+      -X github.com/kopia/kopia/repo.BuildInfo=#{Utils.git_head}
+      -X github.com/kopia/kopia/repo.BuildVersion=#{version}
+      -X github.com/kopia/kopia/repo.BuildGitHubRepo=kopia/kopia
+    ]
+
+    system "go", "build", *std_go_args(ldflags: ldflags)
+
+    output = Utils.safe_popen_read(bin/"kopia", "--completion-script-bash")
+    (bash_completion/"kopia").write output
+
+    output = Utils.safe_popen_read(bin/"kopia", "--completion-script-zsh")
+    (zsh_completion/"_kopia").write output
+
+    output = Utils.safe_popen_read(bin/"kopia", "--help-man")
+    (man/"man1/kopia.1").write output
```

Comment:
```suggestion
    (man1/"kopia.1").write output
```
This is the same as my previous suggestion, but `man1/"kopia.1"` is equivalent to `man/"man1/kopia.1"`. Using `man1/"kopia.1"` will allow us to be consistent with other formulae like `acmetool`.